TMDbLib/Objects/General/AlternativeTitle.cs

```diff
@@ -12,5 +12,11 @@ public class AlternativeTitle
 
         [JsonProperty("title")]
         public string Title { get; set; }
+        
+        /// <summary>
+        /// The type of title (e.g. working title, DVD title, modern title)
+        /// </summary>
+        [JsonProperty("type")]
+        public string Type { get; set; }
```

Comment:
Can we make this an enum?

Reply:
No I don’t think you can - on the TMDB website this is actually editable as a free text field, not a drop down - there are no fixed values for the property unfortunately.